html/webappapis/scripting/events/event-handler-handleEvent-ignored.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<meta charset="utf-8">
+<title>"handleEvent" property of EventHandler should be ignored</title>
+<link rel="help" href="https://html.spec.whatwg.org/#eventhandler">
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<div id="log"></div>
+<script>
+"use strict";
+
+test(t => {
+  const el = document.createElement("div");
+  el.onmouseenter = {
+    get handleEvent() {
+      t.step(() => {
+        assert_unreached('"handleEvent" property should not be looked up');
+      });
+    },
+  };
+  el.dispatchEvent(new MouseEvent("mouseenter"));
+}, 'plain object "mouseenter" handler');
+
+async_test(t => {
+  const handler = Object.create(Function.prototype, {
+    handleEvent: {
+      get: t.unreached_func('"handleEvent" property should not be looked up'),
+    },
+  });
+  assert_true(handler instanceof Function);
+
+  window.onmessage = handler;
+  t.add_cleanup(() => {
+    window.onmessage = null;
+  });
+
+  window.postMessage({}, "*");
+  step_timeout(t.step_func_done());
```

Comment:
I worry that this might run before the `postMessage()` as it has higher priority usually.

Reply:
I've added 50ms timeout in cc367c3.